package.json

```diff
@@ -824,6 +824,11 @@
         "title": "%cmake-tools.command.cmake.pinnedCommands.run.title%",
         "category": "CMake",
         "icon": "$(play-circle)"      
+      },
+      {
```

Comment:
Please add `"when": "cmake:enableFullFeatureSet"`

Reply:
Done.
Just because i'am curious, what is the effect of "enableFullFeatureSet"?

Reply:
Basically, we have a multi-step activation, which helps ensure that we only activate our extension when people want it activated. This `enabledFullFeatureSet` allows us to control when commands appear, so they only appear when we are fully activated.

Reply:
Ok.  Thanks for the explanation!